### PR TITLE
Add tests topicblog form

### PR DESCRIPTION
**Pull request by mcmikashi**
Implement functional test for topicblog form  with selenium

### CODE REVIEW

**Comment by Shriukan33** [comment]
As a general feeling, you should add some comments to structure the code and explain your intents, it would be easier to find your way inside it. 

It's a pretty long file, and there isn't a docstring and very sparse comments, it makes it harder to understand.

Will expand on the review later :)

**Comment by JeffAbrahamson** [comment]
I don't know selenium well, but LGTM overall.

File: transport_nantes/topicblog/tests_topic_blog_form.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        self.selenium.find_element_by_link_text("Contenu (2)").click()
+        body_text_3_md_input = self.selenium.find_element_by_id(
+            "id_body_text_3_md")
+        body_text_3_md_input.send_keys("body 3")
+        self.selenium.find_element_by_name("sauvegarder").click()
+        self.selenium.find_element_by_xpath(
+            "//input[@value='Publier']").click()
+
+    def test_user_permited_create_auto_publish(self):
+        self.selenium.get('%s%s' % (self.live_server_url, '/tb/admin/t/new'))
```
Comment (Shriukan33) [inline transport_nantes/topicblog/tests_topic_blog_form.py:101]: Use a reverse() function, as the path is subject to changes, it's safer to use the name given to the url.

**Review by Shriukan33 — CHANGES_REQUESTED** [review]

**Review by Shriukan33 — COMMENTED** [review]
Seems that Django CI finds errors in the test run

**Comment by Shriukan33** [comment]
LGTM

**Pull request merged**